packages/lite-insight/src/attribution/multiDimension.ts

```diff
@@ -0,0 +1,136 @@
+import { Datum } from '../interface';
+
+type disassembleInfoSub = {
+  startValue: number;
+  endValue: number;
+  diff: number;
+  contribution: number;
+};
+
+type disassembleInfo = {
+  total: number;
+  detail: Object;
+};
+
+const enumerateAllDimensionCombinationsByDFS = (
+  item: Datum,
+  index: number,
+  dimensions: string[],
+  result: Object,
+  stack: string[],
+  measure: string,
+  baseDimension: string,
+  basePointLeft: string | number,
+  basePointRight: string | number
+) => {
```

Comment:
1. Notes could be added to this function and its parameters
2. This function has too many parameters, wrap them as an object may be better.
3. Try use more specific type declaration rather than Object for "result"
4. Return type declaration could be added?

Reply:
5. Return type would be declared in "ReadMe" later.
Other issues are fixed.